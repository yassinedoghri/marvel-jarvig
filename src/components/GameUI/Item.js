import styled from "styled-components";

const Item = styled.li`
  display: inline;
  font-size: 1.2rem;

  &:last-child {
    margin-left: 1em;
  }
`;

export default Item;
