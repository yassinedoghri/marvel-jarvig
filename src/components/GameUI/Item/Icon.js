import styled from "styled-components";


const Icon = styled.span`
    color: ${props => props.theme.colors.dark.primary};
    padding: 0 .6em;
    font-size: 1.8rem;
`;

export default Icon;
