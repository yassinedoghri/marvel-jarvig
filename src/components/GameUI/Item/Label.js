import styled from "styled-components";


const Label = styled.span`
    font-family: Bangers;
    letter-spacing: .1em;
    text-shadow:
       3px 3px 0 #222,
     -1px -1px 0 #222,  
      1px -1px 0 #222,
      -1px 1px 0 #222,
       1px 1px 0 #222;
`;

export default Label;
