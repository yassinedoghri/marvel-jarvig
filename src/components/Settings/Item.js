import styled from "styled-components";

const Item = styled.div`
  position: relative;
`;

export default Item;
