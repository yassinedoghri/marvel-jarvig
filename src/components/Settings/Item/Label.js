import styled from "styled-components";

const Label = styled.p`
    display: inline-block;
    margin-bottom: .5em;
`;

export default Label;
