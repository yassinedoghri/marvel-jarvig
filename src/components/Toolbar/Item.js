import styled from "styled-components";

const Item = styled.button`
  background: transparent;
  border: none;
  color: ${props => props.theme.colors.accent.base};
  font-size: 1.3rem;
  padding: ${props => props.theme.spaces.xs}em
    ${props => props.theme.spaces.sm}em;

  &:hover {
    cursor: pointer;
    color: ${props => props.theme.colors.accent.light};
  }

  &:active {
    color: ${props => props.theme.colors.accent.dark};
  }
`;

export default Item;
