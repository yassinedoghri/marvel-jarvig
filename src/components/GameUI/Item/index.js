import styled from "styled-components";

import Icon from "components/GameUI/Item/Icon";
import Label from "components/GameUI/Item/Label";

const Item = styled.li`
  display: inline;
  font-size: 1.2rem;

  &:last-child {
    margin-left: 1em;
  }
`;

Item.Icon = Icon;
Item.Label = Label;

export default Item;
