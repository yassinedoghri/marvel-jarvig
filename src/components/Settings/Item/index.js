import styled from 'styled-components';

import Label from './Label';
import Counter from './Counter';

const Item = styled.div`
    position: relative;
`;

Item.Label = Label;
Item.Counter = Counter;

export default Item;
