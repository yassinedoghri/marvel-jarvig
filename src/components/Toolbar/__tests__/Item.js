import Item from "components/Toolbar/Item";
import React from "react";
import { renderWithTheme } from "utils/testHelpers";

describe("Toolbar Item", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(<Item>Item</Item>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
