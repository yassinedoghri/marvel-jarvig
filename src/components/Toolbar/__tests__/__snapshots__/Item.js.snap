// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar Item renders correctly 1`] = `
<button
  className="sc-bdVaJa jsCjUK"
>
  Item
</button>
`;
